bodhi/server/services/graphql.py

```diff
@@ -48,11 +48,37 @@ class Query(graphene.ObjectType):
     """Allow querying objects."""
 
     allReleases = graphene.List(Release)
+    specificReleases = graphene.Field(
```

Comment:
I am not a big fan of `specificReleases` for the name of the query, it is a bit long to type for me :P. @StephenCoady do you have an idea on how we could name that ?

Reply:
Yeah I'd agree. Lets make it descriptive but simple, how about `getRelease` ?